Building TF on archlinux with python3

Hi

Sounds like the python3 building script have some issues:
```
PC% bazel build -c opt //tensorflow/tools/pip_package:build_pip_package
...........
WARNING: Sandboxed execution is not supported on your system and thus hermeticity of actions cannot be guaranteed. See http://bazel.io/docs/bazel-user-manual.html#sandboxing for more information. You can turn off this warning via --ignore_unsupported_sandboxing.
INFO: Found 1 target...
ERROR: /home/zenol/tensorflow/tensorflow/tensorflow/python/BUILD:247:1: Converting to Python 3: tensorflow/python/ops/functional_ops.py failed: 2to3 failed: error executing command bazel-out/host/bin/external/bazel_tools/tools/python/2to3 --no-diffs --nobackups --write --output-dir bazel-out/local_linux-py3-opt/genfiles/python3/tensorflow/python/ops --write-unchanged-files ... (remaining 1 argument(s) skipped): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
Target //tensorflow/tools/pip_package:build_pip_package failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 18.140s, Critical Path: 2.02s
```

With verbose_faillures : 
```
ERROR: /home/zenol/tensorflow/tensorflow/tensorflow/python/BUILD:247:1: Converting to Python 3: tensorflow/python/ops/functional_ops.py failed: 2to3 failed: error executing command 
  (cd /home/zenol/.cache/bazel/_bazel_zenol/67d52188e906c7f900d4635d7d1ff821/tensorflow && \
  exec env - \
  bazel-out/host/bin/external/bazel_tools/tools/python/2to3 --no-diffs --nobackups --write --output-dir bazel-out/local_linux-py3-opt/genfiles/python3/tensorflow/python/ops --write-unchanged-files tensorflow/python/ops/functional_ops.py): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1: 2to3 failed: error executing command 
  (cd /home/zenol/.cache/bazel/_bazel_zenol/67d52188e906c7f900d4635d7d1ff821/tensorflow && \
  exec env - \
  bazel-out/host/bin/external/bazel_tools/tools/python/2to3 --no-diffs --nobackups --write --output-dir bazel-out/local_linux-py3-opt/genfiles/python3/tensorflow/python/ops --write-unchanged-files tensorflow/python/ops/functional_ops.py): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
Target //tensorflow/tools/pip_package:build_pip_package failed to build
INFO: Elapsed time: 30.007s, Critical Path: 29.28s
```

Nb : The python2 build seems to work.

I get a similar error message on OSX 10.11.2 with python 3.5.0:

    $ bazel build -c opt //tensorflow/tools/pip_package:build_pip_package --verbose_failures
    INFO: Found 1 target...
    INFO: From Compiling google/protobuf/src/google/protobuf/util/time_util.cc [for host]:
    google/protobuf/src/google/protobuf/util/time_util.cc:52:18: warning: unused variable 'kMicrosPerMillisecond' [-Wunused-const-variable]
    static const int kMicrosPerMillisecond = 1000;
                     ^
    google/protobuf/src/google/protobuf/util/time_util.cc:56:19: warning: unused variable 'kTimestampFormat' [-Wunused-const-variable]
    static const char kTimestampFormat[] = "%E4Y-%m-%dT%H:%M:%S";
                      ^
    google/protobuf/src/google/protobuf/util/time_util.cc:371:6: warning: unused function 'ToUint128' [-Wunused-function]
    void ToUint128(const Timestamp& value, uint128* result, bool* negative) {
         ^
    google/protobuf/src/google/protobuf/util/time_util.cc:396:6: warning: unused function 'ToTimestamp' [-Wunused-function]
    void ToTimestamp(const uint128& value, bool negative, Timestamp* timestamp) {
         ^
    4 warnings generated.
    INFO: From Compiling google/protobuf/src/google/protobuf/util/internal/utility.cc [for host]:
    google/protobuf/src/google/protobuf/util/internal/utility.cc:50:19: warning: unused function 'SkipWhiteSpace' [-Wunused-function]
    const StringPiece SkipWhiteSpace(StringPiece str) {
                      ^
    1 warning generated.
    ERROR: /Users/zeis/tensorflow/tensorflow/python/BUILD:247:1: Converting to Python 3: tensorflow/python/ops/functional_ops.py failed: 2to3 failed: error executing command 
      (cd /private/var/tmp/_bazel_zeis/137871672d8bee8042ab5135b1863c28/tensorflow && \
      exec env - \
      bazel-out/host/bin/external/bazel_tools/tools/python/2to3 --no-diffs --nobackups --write --output-dir bazel-out/local_darwin-py3-opt/genfiles/python3/tensorflow/python/ops --write-unchanged-files tensorflow/python/ops/functional_ops.py): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1: 2to3 failed: error executing command 
      (cd /private/var/tmp/_bazel_zeis/137871672d8bee8042ab5135b1863c28/tensorflow && exec env - bazel-out/host/bin/external/bazel_tools/tools/python/2to3 --no-diffs --nobackups --write --output-dir bazel-out/local_darwin-py3-opt/genfiles/python3/tensorflow/python/ops --write-unchanged-files tensorflow/python/ops/functional_ops.py): com.google.devtools.build.lib.shell.BadExitStatusException: Process exited with status 1.
    Target //tensorflow/tools/pip_package:build_pip_package failed to build
    INFO: Elapsed time: 7.380s, Critical Path: 7.22s

--- a/tensorflow/python/BUILD
+++ b/tensorflow/python/BUILD
@@ -263,6 +263,7 @@ tf_gen_op_wrapper_py(
 py_library(
     name = "functional_ops_lib",
     srcs = ["ops/functional_ops.py"],
+    srcs_version = "PY2AND3",
     deps = [
         ":functional_ops",
     ],

